src/hooks/useVaultRouter.ts

```diff
@@ -65,6 +66,13 @@ export function useVaultRouter(contracts?: Contracts) {
         chainId: contracts.chainId,
       }
     );
+    if (txResult?.tx)
```

Comment:
is the db vault trx record just for display ? I think this part of the code will not execute if the user closes the window/tab before the transaction is mined ?

Reply:
Yeah, it's just for display. Not sure of a better way to do this